src/views/ChatView.vue

```diff
@@ -166,4 +221,25 @@ export default {
 <style scoped lang="stylus">
 .message-input
   margin: 0
+.q-table
+  border: 10px solid rgb(52,73,94)
+.q-list
```

Comment:
be careful with styles like this that might impact someone else using a list the next time. Same potential issue with `button` and `.q-popover`